src/H5C.c

```diff
@@ -3566,575 +3566,575 @@ H5C_unprotect(H5F_t *f, haddr_t addr, void *thing, unsigned flags)
 
 #if H5C_DO_EXTREME_SANITY_CHECKS
     if ((H5C_validate_protected_entry_list(cache_ptr) < 0) ||
-        (H5C_validate_pinned_entry_list(cache_ptr) < 0) || (H5C_validate_lru_list(cache_ptr) < 0)) {
+        (H5C_validate_pinned_entry_list(cache_ptr) < 0) || (H5C_validate_lru_list(cache_ptr) < 0))
```

Comment:
Removing this bracket is the only change I made. Everything else was from the formatter.